src/page/order-summary.page.ts

```diff
@@ -0,0 +1,13 @@
+import { $, ElementFinder } from 'protractor';
+
+export class OrderSummaryPage {
+  private orderSuccessfulConfirm: ElementFinder;
+
+  constructor () {
+    this.orderSuccessfulConfirm = $('#center_column > div > p > strong');
+  }
+
+  public async getOrderSuccessfulConfirm(): Promise<string> {
```

Comment:
we can change this name to mention that we are getting a message or title, like: getConfirmationMessage or getOrderSummaryTitle

Reply:
Done